src/screens/ProfileScreen/index.js

```diff
@@ -708,6 +708,9 @@ const ProfileScreen = ({route}) => {
               source={SOURCE_MY_PROFILE}
               hideThreeDot={false}
               showAnonymousOption={true}
+              onDeletePost={() => onDeletePost()}
+              isSelf={item.is_self}
+              isShowDelete={true}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `onDeletePost` prop has been added to the component and `isShowDelete` has been set to true. Ensure that these changes align with the intended functionality of the component.
```diff
+ onDeletePost={() => onDeletePost()}
+ isSelf={item.is_self}
+ isShowDelete={true}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->